go.mod

```diff
@@ -10,6 +10,7 @@ replace (
 
 require (
 	github.com/ghodss/yaml v1.0.0
+	github.com/gorilla/handlers v1.5.1
```

Comment:
Can you please run `go mod tidy` on this PR?

Reply:
done.